app/controllers/products_controller.rb

```diff
@@ -30,6 +30,11 @@ def update
     end
   end
 
+  def destroy
+    @product.destroy
```

Comment:
make sure that images are removing from s3

Reply:
Removed